content/docs/csidriver/installation/operator/unity.md

```diff
@@ -138,3 +138,38 @@ kubectl edit configmap -n unity unity-config-params
   1. Prior to CSI Driver for unity version 2.0.0, the log level was allowed to be updated dynamically through `logLevel` attribute in the secret object.
   2. "Kubelet config dir path" is not yet configurable in case of Operator based driver installation.
   3. Also, snapshotter and resizer sidecars are not optional to choose, it comes default with Driver installation. 
+
+## Volume Health Monitoring
+This feature is introduced in CSI Driver for unity version 2.1.0.
+
+### Operator based installation
+
+Volume Health Monitoring feature is optional and by default this feature is disabled for drivers when installed via operator.
+To enable this feature, we will have to add the below block to the driver manifest before installing the driver. This ensures to install external health monitor sidecar. To get the volume health state `value` under controller should be set to true as seen below. To get the volume stats `value` under node should be set to true.
```

Comment:
can we also add command "kubectl edit csiunity/unity -n namespace_of_driver" and modify bellow parameters?